python/test/test_property_graph.py

```diff
@@ -350,16 +351,18 @@ def test_types(graph):
         "Post": 11,
     }
     assert graph.node_types.is_subtype_of(0, 1) is True
-    non_atomic_type = graph.get_or_add_non_atomic_entity_type([node_atomic_types["Message"], node_atomic_types["Post"]])
+    non_atomic_type = graph.node_types.get_or_add_non_atomic_entity_type(
+        [node_atomic_types["Message"], node_atomic_types["Post"]]
+    )
     assert (
         graph.node_types.get_non_atomic_entity_type([node_atomic_types["Message"], node_atomic_types["Post"]])
         == non_atomic_type
     )
     assert graph.node_types.get_atomic_subtypes(non_atomic_type) == set(
         [node_atomic_types["Message"], node_atomic_types["Post"]]
     )
-    assert graph.node_types.get_supertypes(node_atomic_types["Message"]) == set(
-        [node_atomic_types["Message"], non_atomic_type]
+    assert graph.node_types.get_supertypes(node_atomic_types["Message"]).issuperset(
+        set([node_atomic_types["Message"], non_atomic_type])
```

Comment:
NIT: You shouldn't need the set constructor here. You can pass any collection, I think. Also Set literals are supported in python. They look like this: `{a, b, c}`. They look a LOT like dict literals though (only differing by the colons), so I kinda prefer the constructor form anyway.